components/homme/src/common_movie_mod.F90

```diff
@@ -133,30 +133,32 @@ module common_movie_mod
                                                  'ilev        ',&
                                                  'nelem       ',&
                                                  'time        ',&
-                                                 'nsubelements'/)  
+                                                 'nsubelements'/)
 #else
-  integer, parameter :: varcnt = 10
+  integer, parameter :: varcnt = 12
   integer, parameter :: maxdims=4
-  character*(*),parameter::dimnames(maxdims)=(/'ncol ','nlev ','nelem','time '/)  
+  character*(*),parameter::dimnames(maxdims)=(/'ncol ','nlev ','nelem','time '/)
   integer, parameter :: vardims(maxdims,varcnt) =  reshape( (/ 1,4,0,0,  &  !ps
                                                                1,2,4,0,  &  !geop
                                                                1,2,4,0,  &  !u
                                                                1,2,4,0,  &  !v
-                                                               1,0,0,0,  &  !lon
-                                                               1,0,0,0,  &  !lat
+                                                               1,0,0,0,  &  !lon (x for planar)
+                                                               1,0,0,0,  &  !lat (y for planar)
                                                                4,0,0,0,  &  ! time
                                                                1,2,4,0,  &  ! zeta
                                                                1,2,4,0,  &  ! div
+                                                               1,2,4,0,  &  ! eta
```

Comment:
what is eta in here?

Reply:
eta is the absolute vorticity (sum of "planetary" and relative vorticity). I will add a comment to this effect. I am also open to changing the name, eta is just the one I am familiar with